QEfficient/utils/constants.py

```diff
@@ -10,7 +10,41 @@
 UTILS_DIR = os.path.dirname(os.path.abspath(__file__))
 QEFF_DIR = os.path.dirname(UTILS_DIR)
 ROOT_DIR = os.path.dirname(QEFF_DIR)

```

Comment:
We should remove these too, as these again create files in the installation path.

Reply:
There is some dependency on other scripts for ROOT_DIR, would like to have this until fully tested.

Reply:
as mentioned in #68 same goes here too, this requires changes in many places making the review difficult, will take this up once this is closed. thanks, resolving the chat.